tests/test_backend.py

```diff
@@ -324,8 +334,14 @@ def setUp(self):
     def tearDown(self):
         # Empty outbox everytime test finishes
         FakeSESConnection.outbox = []
-
+    
+    def test_from_email(self):
+        settings.AWS_SES_FROM_EMAIL = "return@example.com"
+        send_mail('subject', 'body', 'from@example.com', ['to@example.com'])
+        self.assertEqual(self.outbox.pop()['Source'], 'return@example.com')
+    
     def test_return_path(self):
+        settings.USE_SES_V2 = True
         settings.AWS_SES_RETURN_PATH = "return@example.com"
         send_mail('subject', 'body', 'from@example.com', ['to@example.com'])
-        self.assertEqual(self.outbox.pop()['Source'], 'return@example.com')
+        self.assertEqual(self.outbox.pop()['FeedbackForwardingEmailAddress'], 'return@example.com')
```

Comment:
This is the mentioned test/scenery in https://github.com/django-ses/django-ses/pull/276/files#r1169177827

```suggestion
        self.assertEqual(self.outbox.pop()['Source'], 'from@example.com')
        self.assertEqual(self.outbox.pop()['FeedbackForwardingEmailAddress'], 'return@example.com')
```

The value of `settings.AWS_SES_RETURN_PATH` was moved from `Source` to ` FeedbackForwardingEmailAddress`!

Reply:
Use the correct assert then

> self.assertEqual(message['FromEmailAddress'], '[my_default_from@example.com](mailto:my_default_from@example.com)')

Reply:
Does this change need to be added still @peidrao @rodrigondec ?